build.sh

```diff
@@ -0,0 +1,73 @@
+#!/usr/bin/env bash
+
+(
```

Comment:
Why this extra-long block spawning the whole file?

Reply:
That's so the `cd` commands don't affect the calling context (otherwise you might end up in a different directory than your pwd before starting the script)

Reply:
Unless the caller sources this script it's executed as a sub-process anyway, thus no need for starting yet another sub-shell here IMHO.